### ISSUE TITLE
Feature Request: Fill ggridges with Images

### ISSUE THREAD

**Issue by hunterkempf** [wilkelab/ggridges#61]
My Feature Request is to make the fill "color" of the ridges into an image. The idea is similar to this [repo](https://github.com/RobertMyles/flagfillr) (flags on country shapes) or this [repo](https://github.com/gsimchoni/ggwithimages) (images split by line charts or bar charts). I am not sure how hard it would be to incorporate but I think it has the potential to make some cool graphics especially with country level data.

**Comment by trevorld** [comment]
* FYI: I think this should be possible now after https://github.com/wilkelab/ggridges/pull/98 
* You need R 4.2+ and `ggplot2::check_device("patterns")` must be `TRUE` in your active graphics device i.e. the newish pattern fill functionality must be supported by your graphics device.
* You could manually wrap the image in `grid::rasterGrob()` and then in a `grid::pattern()` object or perhaps alternately use `gridpattern::patternFill("image")`.

**Comment by clauswilke** [comment]
Yes. Here is a long tutorial on pattern fills: https://wilkelab.org/dataviz_shortcourse/materials/positconf2024-pattern-fill.html

This should work now with the development version of ggridges, though I haven't tried.
